pkg/provisioner/resourcehandlers_test.go

```diff
@@ -37,11 +37,22 @@ func TestNewCredentialsSecret(t *testing.T) {
 		authSecret   = "test-auth-secret"
 	)
 
+	var T = true
```

Comment:
isn't upper case `T` used to represent a generic type?

Reply:
ah, right. will fix